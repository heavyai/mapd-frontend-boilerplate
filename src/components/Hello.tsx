import React from 'react'


type Props = {
  name: string
}


const Hello: React.StatelessComponent<Props> = ( { name } ) =>
  <div>
    Hello { name }!
  </div>


export default Hello
