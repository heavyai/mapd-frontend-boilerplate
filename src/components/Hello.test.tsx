import React from 'react'
import * as renderer from 'react-test-renderer'

import Hello from './Hello'


test( 'Component renders', () => {
  const component = renderer.create(
    <Hello name='world' />
  )

  expect( component.toJSON() ).toMatchSnapshot()
} )
