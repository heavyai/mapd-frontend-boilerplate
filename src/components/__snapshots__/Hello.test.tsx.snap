// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders 1`] = `
<div>
  Hello 
  world
  !
</div>
`;
